enterprise_access/apps/api/v1/views/bffs/checkout.py

```diff
@@ -41,6 +51,8 @@ def get_permissions(self):
         and get either a hydrated user object from the JWT, or an AnonymousUser
         if not authenticated.
         """
+        if self.action in self.AUTHENTICATED_ONLY_ACTIONS:
+            return [IsAuthenticated()]
```

Comment:
This is the only way I could quickly figure out to have auth'd/unauth'd viewset actions contained within the same URL route.